pages/about/contributing-in-github.md

```diff
@@ -0,0 +1,119 @@
+---
+title: Contributing in GitHub
+search_exclude: true
+description: Instructions on how to contribute content to the DS Handbook in GitHub
+contributors: Marina Popleteeva, Diana Pilvar, Niclas Jareborg, Helena SchnitSchnitzer, Jeanne Wilbrandt
+page_id: contributing_in_github
+---
+
+# Contributing in GitHub
+
+This instruction has been inspired by [RDMkit](https://rdmkit.elixir-europe.org/github_way).
+
+This guide tells you how to create a chapter and edit existing ones using GitHub.
+
+Prerequisites:
+
+- Basic knowledge of Markdown. All you need to know is in the RDMkit [markdown cheat sheet](https://rdmkit.elixir-europe.org/markdown_cheat_sheet)
+
+- A GitHub account. If you do not have one,[ ](https://github.com/join)[create a free GitHub account](https://github.com/join) before you start.
+
+## **Read the guides**
```

Comment:
Maybe "Read writing style/format guides" or "Read relevant guides" ?

Reply:
Read the relevant guides